src/main/groovy/com/lesfurets/jenkins/unit/InterceptingGCL.groovy

```diff
@@ -12,6 +12,9 @@ class InterceptingGCL extends GroovyClassLoader {
         metaClazz.static.invokeMethod = helper.getMethodInterceptor()
         metaClazz.methodMissing = helper.getMethodMissingInterceptor()
         metaClazz.getEnv = {return binding.env}
+        binding.variables.forEach { String property, Object value ->
```

Comment:
@nestoracunablanco, firs of all thank you for the Pull request.

Did you try to use propertyMissing interceptor? 
I'm corious what if you create an instance of library class and then change the binding. 
Could you please add a test to this scenario?

Reply:
Hi @stchar , thanks for the nice feedback :)
To your question, yes, I tried to use `propertyMissing` interceptor, but since the instantiation relies on `PogoMetaClassGetPropertySite` instead of a `PropertyMissingException` a `GroovyRuntimeException` is thrown. I do not know if this is an intentional design decision or a bug, but this is the pragmatic workaround I found.

Reply:
Ok, I see. I tried to implement  `propertyMissing` and `propertyInterceptor` few month ago but wasn't succeed. This groovy interceptor magic is hard)) Glad you  made it worked.